csharp/lib/Cargo.toml

```diff
@@ -18,5 +18,6 @@ tokio = { version = "^1", features = ["rt", "macros", "rt-multi-thread", "time"]
 logger_core = {path = "../../logger_core"}
 
 [profile.release]
-lto = true
-debug = true
+lto = "fat"
```

Comment:
I'm not against using "fat", but end-to-end benchmarks would be nice to ensure we don't incur a performance penalty from doing this.